rllib/examples/evaluation/evaluation_parallel_to_training.py

```diff
@@ -74,7 +74,7 @@ def on_train_result(self, *, algorithm, result, **kwargs):
                 # Compare number of entries in episode_lengths (this is the
                 # number of episodes actually run) with desired number of
                 # episodes from the config.
-                assert num_episodes_done == algorithm.config.evaluation_duration, (
+                assert num_episodes_done + 5 == algorithm.config.evaluation_duration, (
```

Comment:
Ah, is this a bug in the example?

Reply:
This is a test that fails continuously and I added some buffer to test, if this makes a difference in CI tests. It does not. I moved back to the original, but something is not working properly. It will take another look from us.